odoo/addons/base/models/ir_mail_server.py

```diff
@@ -267,17 +291,22 @@ def test_smtp_connection(self):
                     # ignored, just a consequence of the previous exception
                     pass
 
-        message = _("Connection Test Successful!")
         return {
             'type': 'ir.actions.client',
             'tag': 'display_notification',
             'params': {
-                'message': message,
+                'message':
+                    _('Max email size has been set') if autodetect_max_email_size else _('Connection Test Successful!'),
```

Comment:
Maybe we could always say "Connection Succesful.", then add "Email maximum size updated" is the boolean was set ?